middleware.test.ts

```
@@ -0,0 +1,652 @@
+    expect(process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY).toBeDefined();
+  });
+});
+
+// Export test utilities for reuse in other test files
+export const middlewareTestUtils = {
+  createMockNextRequest,
+  createMockNextResponse,
+  mockEnv,
+};
+
+/**
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)